tpke/src/ciphertext.rs

```diff
@@ -2,9 +2,9 @@ use std::ops::Mul;
 
 use ark_ec::{pairing::Pairing, AffineRepr};
 use ark_ff::{One, UniformRand};
-use ark_serialize::{CanonicalSerialize, Compress};
+use ark_serialize::CanonicalSerialize;
 use chacha20poly1305::{
-    aead::{generic_array::GenericArray, Aead, KeyInit},
+    aead::{generic_array::GenericArray, Aead, KeyInit, Payload},
```

Comment:
Is `Payload` your preferred name at this point?

Reply:
That's from the aead crate
https://docs.rs/aead/latest/aead/struct.Payload.html